core/src/main/java/com/github/mizool/core/password/Pbkdf2WithHmacSha1Hasher.java

```diff
@@ -16,57 +16,17 @@
  */
 package com.github.mizool.core.password;
 
-import java.security.NoSuchAlgorithmException;
-import java.security.SecureRandom;
-import java.security.spec.InvalidKeySpecException;
-import java.security.spec.KeySpec;
-import java.util.Base64;
-import java.util.List;
-import java.util.Random;
-
-import javax.crypto.SecretKeyFactory;
-import javax.crypto.spec.PBEKeySpec;
-
-import lombok.Data;
 import lombok.extern.slf4j.Slf4j;
 
+import org.kohsuke.MetaInfServices;
+
 import com.github.mizool.core.configuration.Config;
-import com.github.mizool.core.exception.CodeInconsistencyException;
-import com.google.common.base.Joiner;
-import com.google.common.base.Splitter;
 
 @Slf4j
-public class Pbkdf2WithHmacSha1Hasher implements PasswordHasher
+@Deprecated
```

Comment:
The Javadoc `@deprecated` tag is missing (see [SonarCloud issue](https://sonarcloud.io/project/issues?id=com.github.mizool%3Amizool&issues=AXhKZ-72BOoP0w3KeMvg&open=AXhKZ-72BOoP0w3KeMvg&pullRequest=183)).

This gives us an opportunity to explain _why_ this is deprecated. How about something like this? `For newly saved passwords, use {@link Pbkdf2WithHmacSha512Hasher} instead. This hasher is kept to allow continued verification of old passwords.`